aws/templates/init_script.tftpl

```diff
@@ -59,21 +59,21 @@ systemctl start docker_compose
 
 # Deploy prometheus node exporter
 SCRIPT=/tmp/deploy_node_exporter.sh
-cat > "$SCRIPT" <<EOF
+cat > "$SCRIPT" <<'EOT'
```

Comment:
changing to `EOT` (End-Of-Template) so it doesn't conflict with `EOF`s inside the template.